openacademy/__init__.py

```diff
@@ -1 +1 @@
-from . import models
+from . import models, wizard
```

Comment:
Please import wizard in another line

```suggestion
from . import models
from . import wizard
```

This file is for import the folders with .py files, one line for import.

Reply:
So if we import a folder, then one line per import; but if we import different py files we can do it in the same import?

Reply:
We have 2 cases, we import folders/models themselves or we import libraries.

**First case:** [One line per folder](https://github.com/edy1192/openacademy-project/blob/master/openacademy/__init__.py) or [one line per model](https://github.com/edy1192/openacademy-project/blob/master/openacademy/model/__init__.py).

**Second case:** We group the import in one line according to the library and the order suggested in the [imports section](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#32imports) in the guidelines.